client/components/chart/utils.js

```diff
@@ -400,19 +400,23 @@ const showTooltip = ( node, params, d, position ) => {
 				<li class="key-row">
 					<div class="key-container">
 						<span class="key-color" style="background-color:${ getColor( row.key, params ) }"></span>
-						<span class="key-key">${ row.key }:</span>
+						<span class="key-key">${ d[ row.key ].label ? d[ row.key ].label : row.key }:</span>
```

Comment:
Perhaps remove the `:` at the end as per the [original designs](https://automattic.invisionapp.com/d/main#/console/14672444/310038620/preview)?